ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -175,7 +178,7 @@ class SqlLedgerSpec
       packages: List[DamlLf.Archive],
   ): Future[Ledger] = {
     metrics.getNames.forEach(name => { val _ = metrics.remove(name) })
-    val ledger = newLoggingContext { implicit logCtx =>
+    val ledger = newLoggingContext { implicit loggingContext =>
```

Comment:
There's already `LoggingContext.ForTesting` up there, no need to create one here.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6924/commits/58b9029f6d8a79b1c47b96a0027e0f15bd7ef992